### PR TITLE
Update bundler-audit to v0.5.0

### PR DESCRIPTION
**Pull request by dblandin**
- Update bundler-audit to v0.5.0
- Use new `Advisory#cve_id` and `Advisory#osvdb_id` methods

@codeclimate/review :mag_right:

### CODE REVIEW

File: lib/cc/engine/bundler_audit/unpatched_gem_issue.rb [hunk 1]
```diff
@@ -87,10 +87,7 @@ def solution
         end
 
         def identifier
-          case
-          when advisory.cve then "CVE-#{advisory.cve}"
-          when advisory.osvdb then advisory.osvdb
-          end
+          advisory.cve_id || advisory.osvdb_id
```
Comment (GordonDiggs) [inline lib/cc/engine/bundler_audit/unpatched_gem_issue.rb]: does `cve_id` include "CVE-"`?

Comment (GordonDiggs) [inline lib/cc/engine/bundler_audit/unpatched_gem_issue.rb]: Not a huge deal either way - this isn't used for the fingerprint

**Comment by GordonDiggs** [comment]
LGTM

File: lib/cc/engine/bundler_audit/unpatched_gem_issue.rb (hunk 1 above)
Comment (dblandin) [inline lib/cc/engine/bundler_audit/unpatched_gem_issue.rb]: Yep, the new methods include the `CVE-`and `OSVDB-` prefixes

**Pull request merged**